apollo-router/src/uplink/license_stream.rs

```diff
@@ -533,7 +533,7 @@ mod test {
                 }
                 Event::UpdateLicense(_) => SimpleEvent::UpdateLicense,
                 Event::NoMoreLicense => SimpleEvent::NoMoreLicense,
-                Event::Reload => SimpleEvent::ForcedHotReload,
+                Event::Reload | Event::RhaiReload => SimpleEvent::ForcedHotReload,
```

Comment:
Reviewers: I'm not 100% certain that this is the best thing to do here, but it didn't seem worth creating a new `SimpleEvent` for the new `RhaiReload` event. Happy to change this though if required.

Reply:
I like it